.github/workflows/build_wheels.yml

```diff
@@ -36,15 +58,16 @@ jobs:
     # The manylinux image we use to build the wheels can't install the wheels since it's too old.
     # For this reason we test them separately
     runs-on: ubuntu-latest
-    container:
-      image: python:${{ matrix.python_version }}
     needs: build_bodo_linux_wheels
     steps:
       - uses: actions/download-artifact@v4
         id: download-artifact
         with:
           pattern: cibw-wheels-ubuntu-*
           path: .
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python_version }}
```

Comment:
I did this because I couldn't add `set -exo pipefail` inside the docker container. Thus, the job kept passing even though the test failed. This way, the job fails is the test fails